core/modules/filters/parsedate.js

```diff
@@ -0,0 +1,38 @@
+/*\
+title: $:/core/modules/filters/parsedate.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator converting different date formats into TiddlyWiki's date format
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Export our filter function
+*/
+exports.parsedate = function(source,operator,options) {
+    var results = [];
+    source(function(tiddler,title) {
+        var date = null;
+        switch (operator.operand) {
+            case "ISO":
+                date = new Date(title);
+                break;
+            default:
+                return ["parseDate: Unkown operator " + operator.operand];
```

Comment:
I think tw system messages need to be translatable. see an example: https://github.com/Jermolene/TiddlyWiki5/blob/aaf1aa3821e470735ac24b2f21af58835bc5d2ba/core/modules/filters/format.js#L34

They are part of [misc.multids](https://github.com/Jermolene/TiddlyWiki5/blob/aaf1aa3821e470735ac24b2f21af58835bc5d2ba/core/language/en-GB/Misc.multids#L31) text file

Reply:
We need to consider carefully how to handle errors here.

We don't currently handle or report errors within filter operators; generally a filter operator encountering an error would just not return any results at all.

Reply:
The `filter` operator returns an error message if the suffix is wrong. In case of this operator, I'd like to give an error message, because the list of possible values is quite short (actually 1 currently). A simple typo would be a hard to debug error, because you would not know if your input is just not parsable or if the operand is wrong.

Actually, I'd prefer an out-of-band method to give the user the error message. Like logging the error and returning an empty value. But this would be a new endeavour altogether.

Reply:
> The `filter` operator returns an error message if the suffix is wrong.

Ouch! I hadn't noticed that. On that basis your current implementation makes sense.